test/main.spec.ts

```diff
@@ -76,6 +76,7 @@ test('Get separate env config', async () => {
     // Basic authentication
     process.env['JF_USER'] = 'user';
     process.env['JF_PASSWORD'] = 'password';
+    process.env['JF_ACCESS_TOKEN'] = ''
```

Comment:
The cleanup of JF_USER + JF_PASSWORD + JF_ACCESS_TOKEN might be added to the beforeEach